app/models/concerns/sufia/solr_document_behavior.rb

```diff
@@ -7,6 +7,14 @@ module SolrDocumentBehavior
     include Sufia::SolrDocument::Export
     include Sufia::SolrDocument::Characterization
 
+    def all_titles
```

Comment:
@mjgiarlo Yes I used title first and broke about 29 tests. I renamed them to focus on works only and not break other uses of solr title.